CONTRIBUTING.rst

```diff
@@ -185,6 +185,13 @@ finding, or that have similar ideas for changes or additions.  Send a message
 to the `Zephyr-devel mailing list`_ to introduce and discuss your idea with
 the development community.
 
+Please note that it's common practice on IRC to be away from the
+channel, but still have a client logged in to receive traffic. If you
+ask a question from a particular person and they don't answer, **try
+to stay signed in to the channel** if you can, so they have time to
+respond to you. This is especially important given the many different
+timezones Zephyr developers live in.
```

Comment:
Maybe add something like:

    If you don't get a timely response on IRC, try sending a
    message to the mailing list instead.